### PR TITLE
added disabled buttons

### PR DESCRIPTION
**Pull request by zahra-abdullah**
Issue:

### CODE REVIEW

**Comment by zahra-abdullah** [comment]
https://github.com/shahednasser/sbuttons/pull/291

File: src/components/_basic.less [hunk 1]
```diff
@@ -44,6 +44,16 @@ a.sbtn {
   border-radius: 25px;
 }
 
+button:disabled {
```
Comment (shahednasser) [inline src/components/_basic.less:47]: Please use `.sbtn:disabled` instead of the general `button:disabled`

File: src/components/_basic.less [hunk 2]
```diff
@@ -44,6 +44,16 @@ a.sbtn {
...
+  opacity: 0.55;
+}
+
+a.btn-disabled {
```
Comment (shahednasser) [inline src/components/_basic.less:51]: Please use `.sbtn.btn-disabled` instead of the general `a.btn-disabled`

File: index.html [hunk 3]
```diff
@@ -155,6 +155,27 @@ <h1 class="section-header">How to Use</h1>
                 </div>
               </div>
               <br>
+              <div class="text">
+                To produce disabled buttons, add disabled attribute to the button as shown below.
```
Comment (shahednasser) [inline index.html:159]: Change text to "To make a button disabled, ..."

File: assets/js/buttons-examples.js [hunk 4]
```diff
@@ -173,6 +173,11 @@ $(document).ready(function () {
       classes: "outline-btn",
       children: [],
     },
+    {
```
Comment (shahednasser) [inline assets/js/buttons-examples.js:176]: Place Disabled Buttons in the correct alphabetical position.

**Review by shahednasser — CHANGES_REQUESTED** [review]
Great work! I just have a few comments below.

File: assets/js/buttons-examples.js (hunk 4 above)
Comment (zahra-abdullah) [reply assets/js/buttons-examples.js:181]: Hi @shahednasser !! placed it below animated buttons! Please review! thank you!

File: index.html (hunk 3 above)
Comment (zahra-abdullah) [reply index.html]: requested change is added! Please review! I added example for both button and anchor tags. let me know if thats fine!

File: src/components/_basic.less (hunk 2 above)
Comment (zahra-abdullah) [reply src/components/_basic.less]: common class is added! Please review! thank you!

File: src/components/_basic.less (hunk 1 above)
Comment (zahra-abdullah) [reply src/components/_basic.less]: done! please review @shahednasser ! thank you!

**Comment by shahednasser** [comment]
Great work! There's one thing I noticed which is that the class name is "btn-disabled" but throughout the project we've been using the naming of <new button name>-btn so this affects the consistency. I will add this in a new issue but for now I think you've done a great work so thank you so much!

@all-contributors please add @zahra-abdullah for code and idea

**Comment by shahednasser** [comment]
@all-contributors please add @zahra-abdullah for code and idea

**Pull request merged**